Libraries/PeriphDrivers/Include/MAX32570/MAX32xxx.h

```diff
@@ -48,9 +48,9 @@
 #include "mxc_pins.h"
 #include "mxc_sys.h"
 #include "nvic_table.h"
+#ifdef LIB_BOARD
 #include "board.h"
```

Comment:
Lets decrease number of make parameters.
First I believe the board.h file shall not be include by drivers include file, MAX32xxx.h or mxc.h
It shall not be depend on the board related includes. 

I propose you
- to remove LIB_BOARD  params
- Add board.h file in examples source file(s) and explicitly include it (for example in main.c).

In case of customer have their customer board, they can delete board.h file or they can create it...

Reply:
https://github.com/Analog-Devices-MSDK/msdk/pull/613#issuecomment-1591866010